scripts/gen/tpcc.sh

```diff
@@ -80,14 +80,18 @@ cd "$HOME"
 
 if [ -z "$f_skip_load" ]
 then
-  #./cockroach sql --insecure --url "${pgurls[0]}" -e "
-  # SET CLUSTER SETTING kv.snapshot_recovery.max_rate = '512 MiB';
-  # SET CLUSTER SETTING kv.snapshot_rebalance.max_rate = '512 MiB';
-	# SET CLUSTER SETTING admission.kv.enabled,=false;
-	# SET CLUSTER SETTING admission.sql_kv_response.enabled=false;
-	# SET CLUSTER SETTING admission.sql_sql_response.enabled=false;
-  #";
+  ./cockroach sql --insecure --url "${pgurls[0]}" -e "
+   SET CLUSTER SETTING kv.snapshot_recovery.max_rate = '512 MiB';
+   SET CLUSTER SETTING kv.snapshot_rebalance.max_rate = '512 MiB';
```

Comment:
The above snapshot params don't seem to be of much use as per [1].

[1] https://cockroachlabs.slack.com/archives/C4X2J0RH6/p1643839577157699?thread_ts=1643838665.161849&cid=C4X2J0RH6

Reply:
Removed. thanks!